### DISCUSSION TITLE
I have the code for python 2.6, but isn't this unnecessary anymore?

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#349] [Ideas]
I don't think else's `return (dt.microseconds + (dt.seconds + dt.days * 24 * 3600) * 10**6) / 10**6` will pass anymore.

https://github.com/jertel/elastalert2/blob/master/elastalert/util.py#L265

```
def total_seconds(dt):
    # For python 2.6 compatability
    if dt is None:
        return 0
    elif hasattr(dt, 'total_seconds'):
        return dt.total_seconds()
    else:
        return (dt.microseconds + (dt.seconds + dt.days * 24 * 3600) * 10**6) / 10**6
```

**Comment by jertel** [comment]
I don't think we need to keep Python 2.x compatibility code around any longer. Feel free to remove.

**Reply by nsano-rururu** [reply]
thank you for your answer.
